src/reducers/dataEntryErrorsReducer.js

```diff
@@ -3,6 +3,16 @@ import {
   UPDATE_SERVICE_ERROR,
   DISMISS_DATA_ENTRY_ERRORS,
 } from '../actions';
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: "https://f7a71e0f6282e1a79ba6f604080ec88e@o4507232810631168.ingest.us.sentry.io/4507235352313856",
+  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+  tracePropagationTargets: ["localhost", /^https:\/\/(test.)?gogetta.nyc\/team/],
```

Comment:
IIRC www.gogetta.nyc is also used.
(More generally, any reason to limit the target URLs in the first place? Can't think of any other URLs where this code runs, so is it just that there's no way to specify like `*`?)

Reply:
Fixed in 5559ebbb6adaaaa09ee40e7a70e02f1cc66b06f6

I'm not sure if there's a reason to limit it. This just seems more correct.